server/repository/migrations/sqlite/2021-08-20T10-00_create_store_table/up.sql

```diff
@@ -5,5 +5,6 @@ CREATE TABLE store (
     name_id TEXT NOT NULL,
     code TEXT NOT NULL,
     site_id INTEGER NOT NULL,
+    logo TEXT,
```

Comment:
Only noticed now, this migration file creates the initial store table, and since we already deployed omSupply you need to add a new migration that alters the existing `store` table otherwise this migration wouldn't be applied when updating existing deployments.

1) Add a new migration folder with the current date following the existing date schema (migrations are sorted and executed by name)
2) Alter the `store` table in the new migration file, e.g:
```sql
ALTER TABLE store ADD logo TEXT;
```

Furthermore, you also need to add a matching postgres migration files. To test if it is working, additionally to the sqlite tests, run:
`cargo run --features postgres`
(this runs all test with a local postgres DB which you have to have running on you machine...)